e2e/test/scenarios/embedding/embedding-reproductions.cy.spec.js

```diff
@@ -983,3 +983,39 @@ describe("issue 40660", () => {
     });
   });
 });
+
+describe("issue 47617", () => {
```

Comment:
You can ensure this test works by removing the code below, and the test should fail. I've tried that by the way.